### PR TITLE
docs: add Claude3 ML and RemoteFunc notebooks

### PR DESCRIPTION
**Pull request by GarrettWu**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: notebooks/remote_functions/remote_function_vertex_claude_model.ipynb [hunk 1]
```diff
@@ -0,0 +1,484 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Use BigQuery DataFrames to run Anthropic LLM at scale\n",
+    "\n",
+    "<table align=\"left\">\n",
+    "\n",
+    "  <td>\n",
+    "    <a href=\"https://colab.research.google.com/github/anniexudan/bqtoclaude/blob/main/Python_Notebook_Sample/BigFrames%2BClaude_Remote_Function.ipynb\">\n",
```
[the remark this answers is not in the document]
Comment (GarrettWu) [reply notebooks/remote_functions/remote_function_vertex_claude_model.ipynb:12]: Done

File: notebooks/remote_functions/remote_function_vertex_claude_model.ipynb [hunk 2]
```diff
@@ -0,0 +1,484 @@
...
+    "      <img src=\"https://cloud.google.com/ml-engine/images/colab-logo-32px.png\" alt=\"Colab logo\"> Run in Colab\n",
+    "    </a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a href=\"https://github.com/anniexudan/bqtoclaude/blob/main/Python_Notebook_Sample/BigFrames%2BClaude_Remote_Function.ipynb\">\n",
```
[the remark this answers is not in the document]
Comment (GarrettWu) [reply notebooks/remote_functions/remote_function_vertex_claude_model.ipynb:17]: Done

File: notebooks/remote_functions/remote_function_vertex_claude_model.ipynb [hunk 3]
```diff
@@ -0,0 +1,484 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Use BigQuery DataFrames to run Anthropic LLM at scale\n",
+    "\n",
+    "<table align=\"left\">\n",
+    "\n",
+    "  <td>\n",
+    "    <a href=\"https://colab.sandbox.google.com/github/googleapis/python-bigquery-dataframes/blob/main/notebooks/remote_functions/remote_function_vertex_claude_model.ipynb\">\n",
```
Comment (shobsi) [inline notebooks/remote_functions/remote_function_vertex_claude_model.ipynb:12]: Let's keep it colab.sandbox (also below), colab.research redirects to colab.sandbox internally but colab.sandbox itself isn't accessible publically

Comment (GarrettWu) [reply notebooks/remote_functions/remote_function_vertex_claude_model.ipynb]: Done

**Review by shobsi — APPROVED** [review]

**Pull request merged**